components/cloud-foundry/frontend/src/view/applications/application/services/services.html

```diff
@@ -32,11 +21,32 @@ <h3 translate>There are no services available for this endpoint. Please contact
     </form>
   </div>
 </div>
-
-<div class="panel panel-default"
-  ng-repeat="service in applicationServicesCtrl.services | filter:applicationServicesCtrl.category:true | filter:applicationServicesCtrl.search track by service.metadata.guid">
-  <div class="panel-body service-panel">
-    <service-card service="service" app="applicationServicesCtrl.appModel.application"
-      cnsi-guid="applicationServicesCtrl.cnsiGuid"></service-card>
-  </div>
+<div class="service-manage-message" ng-if="applicationServicesCtrl.managingType">You are currently viewing <b>{{::applicationServicesCtrl.managingType}}</b> instances. <a ng-click="applicationServicesCtrl.filterType = applicationServicesCtrl.ALL_FILTER">Click here</a> to view all types.</div>
```

Comment:
Another one for translating. Will need to pull out the <a> from the translation and pass in the type. So maybe something like..
`<div translate translate-values"{{ applicationServicesCtrl }}">key1</div>
<a translate>key2</a>`

`
"key1": "You are currently viewing {managingType}. "
"key2": "View all types"
`